languages/python/default.nix

```diff
@@ -49,6 +49,79 @@ in {
       defaultText = lib.literalExpression "pkgs: [ ]";
       example = lib.literalExpression "pkgs: [ pkgs.requests ]";
     };
+    config = lib.mkOption {
+      type = with lib.types; nullOr lines;
```

Comment:
Maybe also give the user the option to have a path here. This could just be an alias for a `file` option.